nix/lib.nix

```diff
@@ -27,18 +27,24 @@ rec {
     { pyVersion
     , tahoe-lafs
     }:
-    let
-      tahoe-lafs-package = mach-nix.buildPythonPackage (
-        tahoe-lafs.buildArgs // { python = pyVersion; }
-      );
-    in
-      mach-nix.buildPythonPackage {
-        python = pyVersion;
-        providers.tahoe-lafs = "nixpkgs";
-        overridesPre = [(self: super: {
-          tahoe-lafs = tahoe-lafs-package;
-        })];
+      let
+        tahoe-lafs-package = mach-nix.buildPythonPackage (
+          tahoe-lafs.buildArgs // { python = pyVersion; }
+        );
+      in
+      with pkgs."${pyVersion}Packages"; buildPythonPackage {
         inherit src;
+	pname = "ZKAPAuthorizer";
+        version = "9001";
+        propagatedBuildInputs = [
+          prometheus-client
+          colorama
+          tahoe-lafs-package
+          (callPackage ./compose.nix {})
+          (callPackage ./tahoe-capabilities.nix {})
+          sqlparse
+          autobahn
```

Comment:
This is where we stopped last - `nix build` isn't happy about the version of autobahn we have in this nixpkgs.